pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "setuptools.build_meta"
 name = "any-agent"
 readme = "README.md"
 license = {text = "Apache-2.0"}
-requires-python = ">=3.11"
+requires-python = ">=3.13"
```

Comment:
um. I would not do this and just make it clear that a2a requires 3.13

Reply:
Ooh good idea. I'll do that.